frontend/src/components/layout/navigation/MobileNavigation.tsx

```diff
@@ -0,0 +1,169 @@
+import Link from "next/link";
+import { useLocalization } from "@fluent/react";
+import styles from "./MobileNavigation.module.scss";
+import { SignUpButton } from "./SignUpButton";
+import {
+  Cogwheel,
+  ContactIcon,
+  DashboardIcon,
+  FaqIcon,
+  HomeIcon,
+  NewTabIcon,
+  SignOutIcon,
+  SupportIcon,
+} from "../../Icons";
+import { useRuntimeData } from "../../../hooks/api/runtimeData";
+import { getRuntimeConfig } from "../../../config";
+import { getCsrfToken } from "../../../functions/cookies";
+
+export type MenuItem = {
+  url: string;
+  isVisible?: boolean;
+  icon: JSX.Element;
+  l10n: string;
+};
+
+export type Props = {
+  mobileMenuExpanded: boolean | undefined;
+  hasPremium: boolean;
+  isLoggedIn: boolean;
+  userEmail: string | undefined;
+  userAvatar: string | undefined;
+};
+
+export const MobileNavigation = (props: Props) => {
+  const { mobileMenuExpanded, hasPremium, isLoggedIn, userEmail, userAvatar } =
+    props;
+  const { l10n } = useLocalization();
+  const runtimeData = useRuntimeData();
+  const { supportUrl } = getRuntimeConfig();
+
+  const renderMenuItem = (item: MenuItem) => {
+    const { isVisible = true } = item;
+
+    return isVisible ? (
+      <li className={`${styles["menu-item"]}`}>
+        <Link href={item.url}>
+          <a className={`${styles.link}`}>
+            {item.icon}
+            {l10n.getString(item.l10n)}
+          </a>
+        </Link>
+      </li>
+    ) : null;
+  };
+
+  // We make sure toggle state is not undefined
+  // or we get a flash of the mobile menu on page load.
+  const toggleMenuStateClass =
+    typeof mobileMenuExpanded !== "boolean"
+      ? ""
+      : mobileMenuExpanded
+      ? styles["is-active"]
+      : styles["not-active"];
+
+  return (
+    <nav
+      aria-label={l10n.getString("nav-menu-mobile")}
```

Comment:
Sorry, I missed this before, but this will replace the entire description of the `<nav>`s contents with this label, effectively hiding the menu items inside. The elements themselves and the contents inside should be sufficient to describe this.

```suggestion
```

Reply:
Removed - looks like no errors.